dot/parachain/backing/candidate_backing.go

```diff
@@ -5,124 +5,723 @@ package backing
+			Ch:                      chValidationResultRes,
+		},
+	}
+
+	ValidationResultRes := <-chValidationResultRes
+	if ValidationResultRes.Err != nil {
+		logger.Error(ValidationResultRes.Err.Error())
+	}
+
+	var backgroundValidationResult BackgroundValidationResult
+
+	if ValidationResultRes.Data.IsValid { // Valid
+		// Important: the `av-store` subsystem will check if the erasure root of the `available_data`
+		// matches `expected_erasure_root` which was provided by the collator in the `CandidateReceipt`.
+		// This check is consensus critical and the `backing` subsystem relies on it for ensuring
+		// candidate validity.
+
+		chStoreAvailableDataError := make(chan error)
+		subSystemToOverseer <- parachaintypes.AvailabilityStoreMessage{
+			Value: parachaintypes.ASMStoreAvailableData{
+				CandidateHash: candidateHash,
+				NumValidators: numValidator,
+				AvailableData: parachaintypes.AvailableData{
+					PoV:            pov,
+					ValidationData: pvd,
+				},
+				Ch: chStoreAvailableDataError,
+			},
+		}
+
+		storeAvailableDataError := <-chStoreAvailableDataError
+		switch storeAvailableDataError {
+		case nil:
+			backgroundValidationResult = BackgroundValidationResult{
+				CandidateReceipt:        &candidateReceipt,
+				CandidateCommitments:    &ValidationResultRes.Data.CandidateCommitments,
+				PersistedValidationData: &ValidationResultRes.Data.PersistedValidationData,
+				Err:                     nil,
+			}
+		case ErrInvalidErasureRoot:
+			logger.Debug(ErrInvalidErasureRoot.Error())
+
+			backgroundValidationResult = BackgroundValidationResult{
+				CandidateReceipt: &candidateReceipt,
+				Err:              ErrInvalidErasureRoot,
+			}
+		default:
+			logger.Error(storeAvailableDataError.Error())
+			return
+		}
+
+	} else { // Invalid
+		logger.Error(ValidationResultRes.Data.Err.Error())
+		backgroundValidationResult = BackgroundValidationResult{
+			CandidateReceipt: &candidateReceipt,
+			Err:              ErrInvalidErasureRoot,
+		}
+	}
+
+	chRelayParentAndCommand <- RelayParentAndCommand{
+		RelayParent:   relayPaent,
+		Command:       makeCommand,
+		ValidationRes: backgroundValidationResult,
+		CandidateHash: candidateHash,
+	}
+}
+
+func getPovFromValidator() parachaintypes.PoV {
+	// TODO: Implement this #3545
+	// https://github.com/paritytech/polkadot-sdk/blob/7ca0d65f19497ac1c3c7ad6315f1a0acb2ca32f8/polkadot/node/core/backing/src/lib.rs#L1744 //nolint:lll
+	return parachaintypes.PoV{}
+}
+
+func executorParamsAtRelayParent(
+	relayParent common.Hash, subSystemToOverseer chan<- any,
+) (parachaintypes.ExecutorParams, error) {
+	// TODO: Implement this #3544
+	// https://github.com/paritytech/polkadot-sdk/blob/7ca0d65f19497ac1c3c7ad6315f1a0acb2ca32f8/polkadot/node/subsystem-util/src/lib.rs#L241-L242
+	return parachaintypes.ExecutorParams{}, nil
+}
+
+func (cb *CandidateBacking) processValidatedCandidateCommand(rpAndCmd RelayParentAndCommand) error {
+	// TODO: Implement this
```

Comment:
Are you planning to implement this as part of this PR? Otherwise link the relevant issue here. Or create a new issue.